src/DependencyInjection/Configuration.php

```diff
@@ -134,6 +134,7 @@ protected function addCapsuleSection($node)
                                 ->defaultValue('mysql')
                             ->end()
                             ->scalarNode('host')->defaultValue('localhost')->end()
+                            ->scalarNode('port')->defaultValue(NULL)->end()
```

Comment:
Please use lowercase `null`

Reply:
My fault, netbeans correct to uppercase null here :open_mouth: 
I'm new to github web interface, and never done any testes using phpunit (only junit+dbunit), I can't find a way to fix the fixtures.

Reply:
I would recommend you to check out this project locally and then create branches + PRs instead of the web interface. It's easier to group changes and you can use your favorite IDE/editor :)

In short:

```
# clone this project to WouterJEloquentBundle/
$ git clone https://github.com/wouterj/WouterJEloquentBundle
$ cd WouterJEloquentBundle
# add your fork to the remotes
$ git remote add duayres https://github.com/duayres/WouterJEloquentBundle

# checkout the base branch of your new PR, e.g. master
$ git checkout master
# create a new branch from there named "my-new-branch"
$ git checkout -b my-new-branch
# ... change everything you want to change

# stage all changes for the next commit
$ git add .
# make a commit with your changes
$ git commit -m "My commit message"

# push your changes to your fork
$ git push duayres
# go to github and make a PR!
```